doc/contributor/svm.md

```diff
@@ -19,16 +19,11 @@ The SVM itself, like Graal and TruffleRuby, is implemented in Java.
 
 https://youtu.be/FJY96_6Y3a4?t=10023
 
-More information can be found in Kevin's [blog post](http://nirvdrum.com/2017/02/15/truffleruby-on-the-substrate-vm.html).
+More information can be found in Kevin's
+[blog post](http://nirvdrum.com/2017/02/15/truffleruby-on-the-substrate-vm.html).
 
-To use the SVM you need a release of GraalVM, as described in
-[Using GraalVM](using-graalvm.md).
-
-GraalVM contains a native binary that implements Ruby, similar to the MRI or
-Rubinius executables. By default this binary just starts the JVM, but if you use
-the `--native` flag it runs without a JVM.
-
-The binary doesn't need a JVM:
+The TruffleRuby that is distributed in the [GraalVM](../user/using-graalvm.md)
+by default uses a version compiled using SVM.
```

Comment:
We can't maintain historical information in our documentation like that - for example should I say when each interop semantics changed? The change log covers this requirement.

Reply:
I think it would be fair to make an exception here as it's such a major change and could cause confusion if people have older versions (e.g. they see slow startup on 0.32 but here it says `native` is used if they don't pass any flag). I'll let you judge.

Reply:
Fixing.